controls/cis_al2023.yml

```diff
@@ -0,0 +1,2176 @@
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_etc_passwd
+      - file_owner_etc_passwd
+      - file_permissions_etc_passwd
+
+  - id: 6.1.3
+    title: Ensure permissions on /etc/passwd- are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_backup_etc_passwd
+      - file_owner_backup_etc_passwd
+      - file_permissions_backup_etc_passwd
+
+  - id: 6.1.4
+    title: Ensure permissions on /etc/group are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_etc_group
+      - file_owner_etc_group
+      - file_permissions_etc_group
+
+  - id: 6.1.5
+    title: Ensure permissions on /etc/group- are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_backup_etc_group
+      - file_owner_backup_etc_group
+      - file_permissions_backup_etc_group
+
+  - id: 6.1.6
+    title: Ensure permissions on /etc/shadow are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_owner_etc_shadow
+      - file_groupowner_etc_shadow
+      - file_permissions_etc_shadow
+
+  - id: 6.1.7
+    title: Ensure permissions on /etc/shadow- are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_backup_etc_shadow
+      - file_owner_backup_etc_shadow
+      - file_permissions_backup_etc_shadow
+
+  - id: 6.1.8
+    title: Ensure permissions on /etc/gshadow are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_etc_gshadow
+      - file_owner_etc_gshadow
+      - file_permissions_etc_gshadow
+
+  - id: 6.1.9
+    title: Ensure permissions on /etc/gshadow- are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_backup_etc_gshadow
+      - file_owner_backup_etc_gshadow
+      - file_permissions_backup_etc_gshadow
+
+  - id: 6.1.10
+    title: Audit system file permissions (Manual)
+    levels:
+      - l2_server
+    status: manual
+    related_rules:
+      - rpm_verify_permissions
+      - rpm_verify_ownership
+
+  - id: 6.1.11
+    title: Ensure world writable files and directories are secured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_permissions_unauthorized_world_writable
+      - dir_perms_world_writable_sticky_bits
+
+  - id: 6.1.12
+    title: Ensure no unowned or ungrouped files or directories exist (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - no_files_unowned_by_user
+      - file_permissions_ungroupowned
+
+  - id: 6.1.13
+    title: Ensure SUID and SGID files are reviewed (Manual)
+    levels:
+      - l1_server
+    status: manual
+    rules:
```

Comment:
Should be `related_rules` if you want to let it manual.

Reply:
done.
Also adjusted another 'manual' rule with its related_rules set.
Thanks!